classes/class-ledyer-lco-gateway.php

```diff
@@ -372,14 +372,6 @@ public function add_data_attributes( $tag, $handle ) {
 		 * @param $order
 		 */
 		public function wc_order_created( $order_id, $posted_data, $order ) {
-			$ledyer_order_id = WC()->session->get( 'lco_wc_order_id' );
-
-			$ledyer_order = ledyer()->api->update_order_reference( $ledyer_order_id, array( 'reference' => strval( $order_id ) ) );
-
-			if ( ! $ledyer_order || ( is_object( $ledyer_order ) && is_wp_error( $ledyer_order ) ) ) {
-				// If failed then bail.
-				return;
-			}
```

Comment:
Setting merchant reference here is outdated, since we not should do it when wc order is created, since that now occurs before the order has been created in Ledyer. Ideally we should do it where we receive the notification